js/main.js

```diff
@@ -7,55 +7,88 @@ var $defaultImage = document.querySelector('.default-image');
 
 var $url = document.querySelector('#journal-URL');
 
+var $title = document.querySelector('#journal-title');
+
+var $note = document.querySelector('#journal-notes');
+
+var $dataEntryId = data.nextEntryId;
+
+var $formTitle = document.querySelector('.form-title');
+
 $url.addEventListener('input', function updateImage(event) {
-  var urlPhoto = $form.elements.photo.value;
+  var urlPhoto = $url.value;
   $defaultImage.setAttribute('src', urlPhoto);
 });
 
 function handleSubmit(event) {
   event.preventDefault();
 
-  var $titleValue = $form.elements.title.value;
-  var $urlValue = $form.elements.photo.value;
-  var $notesValue = $form.elements.notes.value;
-  var dataValue = {
-    title: $titleValue,
-    photo: $urlValue,
-    note: $notesValue,
-    nextEntryId: data.nextEntryId
-  };
-  data.entries.unshift(dataValue);
-  data.nextEntryId++;
-  $defaultImage.setAttribute('src', 'images/placeholder-image-square.jpg');
+  if (data.editing === null) {
+    var dataValue = {
+      title: $title.value,
+      photo: $url.value,
+      note: $note.value,
+      entryId: $dataEntryId
+    };
+    $dataEntryId++;
+    data.entries.unshift(dataValue);
+    $ul.prepend(newEntries(dataValue));
+  } else if (data.editing !== null) {
```

Comment:
This can just be an `else`. If the above `if` statement doesn't run, then you can confidently say that `data.editing` is not `null`.